<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Technician</class>
 <widget class="QDialog" name="Technician">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Technician</string>
  </property>
  <widget class="QLabel" name="technician_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Technicien :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="technician_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="contactInformation_GroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Contact</string>
   </property>
   <widget class="QLabel" name="name_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_LineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="phoneNumber_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Téléphone : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNumber_LineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_LineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="email_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Courriel :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="note_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Note :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="note_PlainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>401</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="set_PushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>300</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="inventory_TableWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>51</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inventory_Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventaire :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transfer_PushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfert</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>technician_LineEdit</tabstop>
  <tabstop>name_LineEdit</tabstop>
  <tabstop>phoneNumber_LineEdit</tabstop>
  <tabstop>email_LineEdit</tabstop>
  <tabstop>note_PlainTextEdit</tabstop>
  <tabstop>cancel_PushButton</tabstop>
  <tabstop>set_PushButton</tabstop>
  <tabstop>inventory_TableWidget</tabstop>
  <tabstop>transfer_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
